tests/mock/tags.rs

```diff
@@ -31,8 +31,9 @@ fn test_tags_simple() {
 
     let futcheck = dclient.get_tags(name, None);
 
-    let res = runtime.block_on(futcheck.collect()).unwrap();
-    assert_eq!(res, vec!["t1", "t2"]);
+    let res = runtime.block_on(futcheck.collect::<Vec<_>>());
+    assert_eq!(res.get(0).unwrap().as_ref().unwrap(), &String::from("t1"));
```

Comment:
Can we unpack the steps to avoid the two chained `unwrap`s in a single line? That would improve line-referencing in case of test failures.